modules/ARHParser.jsm.js

```diff
@@ -16,6 +16,7 @@
 // options for ESLint
 /* global Components, Services */
 /* global Logging */
+/* global DKIM_InternalError */
```

Comment:
Note that you can specify multiple variables in one in one of this global statements, Normally I make two groups, one for things coming from Mozilla, and another one the the add-on specific ones.

```suggestion
/* global Logging, DKIM_InternalError */
```

But you can keep it as is if you prefer one line for each.